test/e2e/README.md

```diff
@@ -0,0 +1,93 @@
+## The Example App end-to-end tests
+
+These e2e tests are used to ensure that all web-based implementations of `The Example App` share the same functionality. You may use this repository as a reference for implementing end-to-end tests with [cypress](https://www.cypress.io/). It is not necessary to use this repository to get `The Example App` running on your machine.
+
+## What is Contentful?
+
+[Contentful](https://www.contentful.com) provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
+
+## Setup
+
+### Circle-CI configuration file
+
+These instructions are for Circle CI 1.0 config files. If you are using 2.0 config files, see here: https://github.com/contentful/the-example-app.csharp/blob/master/.circleci/config.yml#L24-L31
+
+#### 1. Add the node runtime to your CI run:
+
+```
+machine:
+  node:
+    version: "8"
+```
+
+#### 2. Download and install the tests:
+
+```
+dependencies:
+  pre:
+    - git clone https://github.com/contentful/the-example-app-e2e-tests.git ./test/e2e
```

Comment:
Do we still need this? since we are shipping the tests with the repo

Reply:
true, i'm not doing this in the config. I'll clean up the readme.